packages/web-app-files/tests/unit/components/SideBar/Versions/FileVersions.spec.ts

```diff
@@ -191,6 +185,14 @@ function getMountedWrapper({
     downloadFile
   }
   mocks.$clientService.webdav.getFileInfo.mockResolvedValue(mock<Resource>({ id: '1' }))
+  mocks.$clientService.webdav.listFileVersions.mockImplementation(() => {
+    if (loadingStateDelay > 0) {
+      return new Promise((res) => setTimeout(() => res(versions), loadingStateDelay))
```

Comment:
if `loadingStateDelay` defaults to 0 anyway - can't we just always run this branch? What benefit does the check add? a tick less? 🤷🏻

Reply:
Good catch, fixed. It's necessary for checking the loading spinner, since we still don't have a way to mock tasks properly.

Reply:
Hmm CI seems unhappy that way, so I reverted it